rfcs/text/0001-wildcard-data-type.md

```diff
@@ -48,46 +81,114 @@ Proposed type change:
 
+Looking at the following example of a stack trace:
+
+```java
+bootstrap method initialization exception
+at java.base/java.lang.invoke.BootstrapMethodInvoker.invoke(BootstrapMethodInvoker.java:194)
+at java.base/java.lang.invoke.CallSite.makeSite(CallSite.java:315)
+at java.base/java.lang.invoke.MethodHandleNatives.linkCallSiteImpl(MethodHandleNatives.java:259)
+at java.base/java.lang.invoke.MethodHandleNatives.linkCallSite(MethodHandleNatives.java:249)
+at org.elasticsearch.client.RestHighLevelClient.parseEntity(RestHighLevelClient.java:1883)
+at org.elasticsearch.client.RestHighLevelClient.lambda$performRequestAndParseEntity$9(RestHighLevelClient.java:1564)
+at org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1628)
+at org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1596)
+at org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1563)
+at org.elasticsearch.client.IndicesClient.getMapping(IndicesClient.java:282)
+
+Caused by: java.lang.invoke.LambdaConversionException: Invalid receiver type interface org.apache.http.Header; not a subtype of implementation type interface org.apache.http.NameValuePair
+at java.base/java.lang.invoke.AbstractValidatingLambdaMetafactory.validateMetafactoryArgs(AbstractValidatingLambdaMetafactory.java:254)
+at java.base/java.lang.invoke.LambdaMetafactory.metafactory(LambdaMetafactory.java:327)
+at java.base/java.lang.invoke.BootstrapMethodInvoker.invoke(BootstrapMethodInvoker.java:127)
+```
+
+If I wanted to look for similar events that also contain the phrase `lambda$performRequestAndParseEntity$9(RestHighLevelClient.java`, I'd need a field that supports searching in the middle of a string. Keyword would perform poorly and text would require rethinking the query to match the analyzer and tokenization applied at index time.
+
 ### Command-line execution
 
-Building accurate queries again a full command line string field like the `process.command_line` field, also benefit from wildcard improvements:
+The arguments, order of those arguments, and values passed can be arbitrary in a command-line execution. If searching across multiple arguments, retaining their ordering, and/or the argument-value pairing is key to the search criteria, multiple wildcards patterns may be needed in a single query (if the presence of the arguments/values is the only criteria regardless of ordering or pairing, using a structured field such as `process.args` would be preferred). Wildcard searching such an unstructured field indexed as keyword, like `process.command_line`, can cause performance challenges.
```

Comment:
Great info. I would have the whole parens on `.args` as a distinct sentence, rather than in parens.